src/App.tsx

```diff
@@ -17,18 +76,32 @@ export const App: React.FC = () => {
             <h1 className="title">Todos:</h1>
 
             <div className="block">
-              <TodoFilter />
+              <TodoFilter
+                getOption={setFilterOption}
+                getSearch={setSearchQuery}
+                clearSearch={setSearchQuery}
```

Comment:
The `clearSearch` prop should be a function that clears the search query. Currently, it directly sets the search query, which might not be the intended behavior. Consider changing it to a function that sets the search query to an empty string.